documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -8,3 +8,4 @@
 
 ### Resolved issues
 * (IDETECT-4056) Resolved a problem with CPAN detector where no components were reported.
```

Comment:
Tweak suggestion:
* (IDETECT-4056) Resolved an issue where no components were reported by CPAN detector.   
<note type="note">If the cpan command has not been previously configured and run on the system, [sollution_name] instructs CPAN to accept default configurations.</note>

Reply:
As per Shanty, the second line may not be 100% required, so I changed it to a note. (Up to you if you want it that way or leave it out.) ;)